lib/SaferpayJson/Request/Container/PayerProfile.php

```diff
@@ -0,0 +1,362 @@
+<?php declare(strict_types=1);
+
+namespace Ticketpark\SaferpayJson\Request\Container;
+
+use JMS\Serializer\Annotation\SerializedName;
+use JMS\Serializer\Annotation\Type;
+
+final class PayerProfile
+{
+    /**
+     * @var bool|null
+     * @SerializedName("HasAccount")
+     */
+    private $hasAccount;
+
+    /**
+     * @var bool|null
+     * @SerializedName("HasPassword")
+     */
+    private $hasPassword;
+
+    /**
+     * @var bool|null
+     * @SerializedName("PasswordForgotten")
+     */
+    private $passwordForgotten;
+
+    /**
+     * @var string|null
+     * @SerializedName("FirstName")
+     */
+    private $firstName;
+
+    /**
+     * @var string|null
+     * @SerializedName("LastName")
+     */
+    private $lastName;
+
+    /**
+     * @var string|null
+     * @SerializedName("Company")
+     */
+    private $company;
+
+    /**
+     * @var string|null
+     * @SerializedName("DateOfBirth")
+     */
+    private $dateOfBirth;
+
+    /**
+     * @var \DateTime|null
+     * @SerializedName("LastLoginDate")
+     * @Type("DateTime<'Y-m-d\TH:i:s.uT'>")
+     */
+    private $lastLoginDate;
+
+    /**
+     * @var string|null
+     * @SerializedName("Gender")
+     */
+    private $gender;
+
+    /**
+     * @var \DateTime|null
+     * @SerializedName("CreationDate")
+     * @Type("DateTime<'Y-m-d\TH:i:s.uT'>")
+     */
+    private $creationDate;
+
+    /**
+     * @var \DateTime|null
+     * @SerializedName("PasswordLastChangeDate")
+     * @Type("DateTime<'Y-m-d\TH:i:s.uT'>")
+     */
+    private $passwordLastChangeDate;
+
+    /**
+     * @var string|null
+     * @SerializedName("Email")
+     */
+    private $email;
+
+    /**
+     * @var string|null
+     * @SerializedName("SecondaryEmail")
+     */
+    private $secondaryEmail;
+
+    /**
+     * @var Phone|null
+     * @SerializedName("Phone")
+     */
+    private $phone;
+
+    /**
+     * @return bool|null
+     */
+    public function getHasAccount(): ?bool
+    {
+        return $this->hasAccount;
+    }
+
+    /**
+     * @param bool|null $hasAccount
+     * @return PayerProfile
+     */
```

Comment:
Method docblocks can be removed everywhere as the code is self-documenting due to type hints.